apps/damap-frontend/src/app/app.component.ts

```diff
@@ -1,10 +1,17 @@
 import { Component } from '@angular/core';
+import { Title } from "@angular/platform-browser";
+import { environment } from "../environments/environment";
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: [],
 })
 export class AppComponent {
-  title = 'Damap Frontend';
+  constructor(private titleService: Title) {}
+
+  ngOnInit(): void {
+    this.titleService.setTitle(environment.title);
+  }
```

Comment:
Minor: Maybe just add a check if the variable is set, and else add a default value.

Reply:
Added the check, although this will only trigger if someone sets the variable to an empty string or undefined/null. Deleting the variable will crash the application. I dont know how to check that, if this is the case you wanted to cover.